docs/visual-basic/getting-started/whats-new.md

```diff
@@ -51,6 +54,42 @@ Bit-shift operators, loop variable declaration
 Visual Basic / Visual Studio .NET 2002   
 The first release of Visual Basic .NET
 
+## Visual Basic 15.5
+
+[Non-trailing named arguments](../programming-guide/language-features/procedures/passing-arguments-by-position-and-by-name#mixing-arguments-by-position-and-by-name)
+
+In Visual Basic 15.3 and earlier versions, when a method call included arguments both by position and by name, positional arguments had to precede named arguments. Starting with Visual Basic 15.5, positional and named arguments can appear in any order as long as the ending positional arguments are in the correct position. This is particularly useful when named arguments are used to make code more readable.
```

Comment:
I think "as long as the ending positional arguments are in the correct position." could be misleading. It would be more accurate to say "as long as all arguments up to the last positional argument are in the correct position." 

The subtle difference is that any named args that appear before positional arguments must be in the correct position.